pkg/osvscanner/osvscanner.go

```diff
@@ -331,6 +333,33 @@ func (m *gitIgnoreMatcher) match(absPath string, isDir bool) (bool, error) {
 	return m.matcher.Match(pathInGitSep, isDir), nil
 }
 
+func scanImage(r reporter.Reporter, path string) ([]scannedPackage, error) {
```

Comment:
What does the table output look like for this?

Reply:
Added a integration test, it looks like this:

```
+--------------------------------+------+-----------+---------+-----------+--------------------------------------------------------------+
| OSV URL                        | CVSS | ECOSYSTEM | PACKAGE | VERSION   | SOURCE                                                       |
+--------------------------------+------+-----------+---------+-----------+--------------------------------------------------------------+
| https://osv.dev/CVE-2018-25032 | 7.5  | Alpine    | zlib    | 1.2.11-r1 | ./fixtures/oci-image/alpine-tester.tar:/lib/apk/db/installed |
| https://osv.dev/CVE-2022-37434 | 9.8  | Alpine    | zlib    | 1.2.11-r1 | ./fixtures/oci-image/alpine-tester.tar:/lib/apk/db/installed |
+--------------------------------+------+-----------+---------+-----------+--------------------------------------------------------------+
```